switch deployer_namespace to kube-system (#32)
spotinst/terraform-spotinst-ocean-spark: README.md

## Migration Guide

### v2 migration guide

> By default our bootstraping job now run in the kube-system namespace

Reviewer: ```suggestion
By default the Ocean Spark deployer jobs now run in the kube-system namespace.
```

> To avoid deletion of your cluster, you will need to set the following line:

Reviewer: ```suggestion
To avoid issues for existing clusters you will need to set the following line:
```
